docs/src/selenium-grid.md

```diff
@@ -33,6 +33,34 @@ You don't have to change your code, just use [`method: BrowserType.launch`] as u
 
 When using Selenium Grid Hub, you can [skip browser downloads](./browsers.md#skip-browser-downloads).
 
+If your grid requires additional capabilities to be set, you can use `SELENIUM_REMOTE_CAPABILITIES` environment variable to provide JSON-serialized capabilities.
```

Comment:
Let's mention that some services that provide hosted selenium grids require additional capabilities.